src/client/redux/clientSideGameExtras/clientSideGameExtras.ts

```diff
@@ -1,17 +1,31 @@
 import { createSlice, PayloadAction, Reducer } from '@reduxjs/toolkit';
+import { Card } from '@/types/cards';
 
 /**
  * Client-side held only game information, to store multiple things like:
  * - the attacking unit
  */
 interface ClientSideGameExtras {
     attackingUnit?: string; // cardId
+    lastPlayedCards?: Card[];
 }
 
+const NUM_LAST_PLAYED_CARDS = 20;
+
 export const clientSideGameExtrasSlice = createSlice({
     name: 'clientSideExtras',
-    initialState: { attackingUnit: undefined },
+    initialState: { attackingUnit: undefined, lastPlayedCards: [] },
     reducers: {
+        receiveLastPlayedCard(state, action: PayloadAction<Card>) {
+            state.lastPlayedCards = [
+                ...state.lastPlayedCards,
+                action.payload,
+            ].splice(-NUM_LAST_PLAYED_CARDS);
+        },
+        startGame(state) {
```

Comment:
instead of resetState or something that sounds computer-y, we prefer actions that resemble human behaviors